packages/alea-frontend/pages/api/nap/get-student-answer.ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { checkIfGetOrSetError, executeAndEndSet500OnError } from '../comment-utils';
+import { Answer, AnswerClass, Grade } from '@stex-react/api';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfGetOrSetError(req, res)) return;
+  const id = req.query.id as string;
```

Comment:
Please add a todo here to do proper access control. Only the student themself and the instructors should be able to view the answers. 
Peers should be able to see the answers if there is a 'peer review request' but in that case we should share the student id.